src/components/MetricsPane/charts/SampleEventBleachingPlot.jsx

```diff
@@ -12,31 +12,27 @@ const bleachingBenthicCategories = Object.entries(
 
 export const SampleEventBleachingPlot = ({ bleachingData }) => {
   const {
-    percent_hard_avg_avg,
-    percent_soft_avg_avg,
-    percent_algae_avg_avg,
-    quadrat_count_avg,
-    sample_unit_count,
+    percent_hard_avg_avg: percentHard,
+    percent_soft_avg_avg: percentSoft,
+    percent_algae_avg_avg: percentAlgae,
+    quadrat_count_avg: quadratCount,
+    sample_unit_count: sampleUnitCount,
   } = bleachingData
 
-  const isBleachingDataAvailable =
-    !!percent_hard_avg_avg &&
-    !!percent_soft_avg_avg &&
-    !!percent_algae_avg_avg &&
-    !!quadrat_count_avg
+  const isBleachingDataAvailable = [percentHard, percentSoft, percentAlgae, quadratCount].every(
+    (value) => value !== null && value !== undefined,
```

Comment:
WAs the previous code my blunder again where I forgot about zeros? this is a nice little function.